solution/index.js

```diff
@@ -0,0 +1,548 @@
+//Edit a task's value in tasks by its ID
+function editTaskValue(taskId, newValue){
+    for(let task of Tasks.todo){
+        if(taskId === task.id){
+            task.task = newValue;
+        }
+    }
+    for(let task of Tasks["in-progress"]){
+        if(taskId === task.id){
+            task.task = newValue;
+        }
+    }
+    for(let task of Tasks.done){
+        if(taskId === task.id){
+            task.task = newValue;
+        }
+    }
+};
+
+//Edit task content
+function editByDblClick(taskElement){
+    let input = document.createElement("input");
+    let taskId = taskElement.id;
+    input.value = taskElement.innerText;
+    //When input loses focus its value is transferd to the task element
+    input.onblur = () => {
+        taskElement.innerText = input.value;
+        //Edits the task's value in the Task object
+        editTaskValue(taskId, input.value);
+        //Storing the change in the local storage
+        saveTasksToStorage();
+    }
+    //Replacing the existent task with the input on dbl-click
+    taskElement.innerText="";
+    taskElement.appendChild(input);
+	input.focus();
+};   
+
+//----Change task list - Hover + alt + 1/2/3----\\
+//Remove task from Tasks object by id and saves to local storage
+function removeTask(taskId) {
+    for(let i = 0; i < Tasks.todo.length; i++){
+        if(Tasks.todo[i].id == taskId){
+            if(Tasks.todo.length === 1 || i === 0){
+                Tasks.todo.shift();
+            }else{
+                Tasks.todo.splice(i, i); 
+            }
+        }
+     }
+     for(let i = 0; i < Tasks["in-progress"].length; i++){
+        if(Tasks["in-progress"][i].id == taskId){
+            if(Tasks["in-progress"].length === 1 || i === 0){
+                Tasks["in-progress"].shift();
+            }else{
+                Tasks["in-progress"].splice(i, i); 
+            }
+        }
+     }
+     for(let i = 0; i < Tasks.done.length; i++){
+        if(Tasks.done[i].id == taskId){
+            if(Tasks.done.length === 1 || i === 0){
+                Tasks.done.shift();
+            }else{
+                Tasks.done.splice(i, i); 
+            }
+        }
+     }
+     document.getElementById(taskId).remove();
+     saveTasksToStorage();
+};
+
+//Removes the task from the current list array and assign it to the given list and creates its element
+function replaceList(taskId, task, newList) {
+    //removes the chosen task from the Tasks object and from the DOM
+    removeTask(taskId);
+    //Creates a task object and places iy in the correct array and list element
+    let taskObject = createTaskObject(taskId, task);
+    createAndPlace(taskObject, newList);
+    Tasks[newList].push(taskObject);
+    //Saves changes to local storage
+    saveTasksToStorage();
+};
+
+//Handles Hover + alt + 1/2/3 over element
+function changeByAlt(e){
+    if(e.type === "mouseover"){
+        if(e.target.className === "task"){
+            document.taskElement = e.target;
```

Comment:
This is practically a global variable. Avoid those - it is a recipe for bugs. There are better ways to pass the relevant element to the future keyboard event handler.